### PR TITLE
Leaflet - Add missing L.ImageOverlay methods and missing L.Draggable class

### PR DESCRIPTION
**Pull request by danzel**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).
- [x] Run `npm run lint -- package-name` if a `tslint.json` is present.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Leaflet/Leaflet/blob/master/src/dom/Draggable.js
https://github.com/Leaflet/Leaflet/pull/5158
- [ ] Increase the version number in the header if appropriate.

### CODE REVIEW

File: leaflet/leaflet-tests.ts [hunk 1]
```diff
@@ -326,3 +326,9 @@ map = map
 	.remove()
 	.whenReady(() => {})
 	.whenReady(() => {}, {});
+
+var elementToDrag = document.createElement('div');
+var draggable = new L.Draggable(elementToDrag);
```
Comment (alejo90) [inline leaflet/leaflet-tests.ts]: Please replace `var` with `const` or `let`.

File: leaflet/index.d.ts [hunk 2]
```diff
@@ -523,6 +538,15 @@ declare namespace L {
...
         setUrl(url: string): this;
+
+        /** Update the bounds that this ImageOverlay covers */
+        setBounds(bounds: LatLngBounds): this;
+
+        /** Get the bounds that this ImageOverlay covers */
+        getBounds(): LatLngBounds;
+
+	    /** Get the img element that represents the ImageOverlay on the map */
+	    getElement(): HTMLImageElement;
```
Comment (alejo90) [inline leaflet/index.d.ts]: Can you please fix the indentation for these two lines.

**Review by alejo90 — CHANGES_REQUESTED** [review]
Thanks for your contribution. We just need a formatting fix and the replacement of `var` with `let` or `const`.

**Comment by danzel** [comment]
Done.

**Comment by alejo90** [comment]
👍

**Comment by alejo90** [comment]
@andy-ms @vvakame

**Pull request merged**